cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -120,6 +129,12 @@ func (m command) build() *cli.Command {
 			Usage:       "the config mode for runtimes that support multiple configuration mechanisms",
 			Destination: &config.mode,
 		},
+		&cli.StringFlag{
```

Comment:
We need to add something like the following to `validateFlags`:
```
	switch config.configSource {
	case configSourceCommand:
		if config.runtime == "docker" {
			m.logger.Warningf("A %v Config Source is not supported for %v; using %v", config.configSource, config.runtime, configSourceFile)
			config.configSource = configSourceFile
		}
	case configSourceFile:
		break
	default:
		return fmt.Errorf("unrecognized Config Source: %v", config.configSource)
	}
```

Reply:
Sure